src/scripting/CompilerTxt.js

```diff
@@ -174,6 +175,10 @@ module.exports = class CompilerTxt extends CompilerBase {
         }
         script += JSON.stringify(set.sourceData, null, 2) + this.eol
       }
+
+      for (const asserter of set.asserters) {
+        script += asserter.name + ' ' + asserter.args.join('|') + this.eol
```

Comment:
for..of loops are not supported by the Node.js version we are targeting (7). please make another kind of loop here